modules/consul-cluster/variables.tf

```diff
@@ -24,6 +24,11 @@ variable "allowed_inbound_cidr_blocks" {
   type        = "list"
 }
 
+variable "allowed_inbound_security_group_ids" {
+  description = "A list of security group IDs that will be allowed to connect to Consul"
+  type        = "list"
```

Comment:
For backwards compatibility, should probably default this to an empty list.

Reply:
I've now set this to default = []